script/tool/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: flutter_plugin_tools
 description: Productivity utils for flutter/plugins and flutter/packages
 repository: https://github.com/flutter/plugins/tree/main/script/tool
-version: 0.9.0
+version: 0.9.1
```

Comment:
This doesn't need a version bump. The tool doesn't follow the versioning rules of the normal packages.

Reply:
fixed this with commit "version bump"(e472173)